lib/bootloader_setup.pm

```diff
@@ -249,7 +250,24 @@ sub boot_local_disk {
         if (match_has_tag 'inst-slof') {
             diag 'specifying local disk for boot from slof';
             my $slof = 5;
-            $slof += 1 if get_var('VIRTIO_CONSOLE');
+            if (get_var('VIRTIO_CONSOLE') && is_ppc64le) {
+                type_string_very_slow "devalias";
+                send_key 'ret';
+                save_screenshot;
+                my $file = './serial0';
+                open (FILE, $file);
+                my @content = <FILE>;
+                close FILE;
+                my $out = 0;
+                foreach my $line (@content) {
+                     if ($line =~ /^disk/) {
```

Comment:
what's diff between "disk" and "scsi" line?

Reply:
disk : /pci@800000020000000/scsi@7

Reply:
@tinawang123 one open question for a possible follow up on this PR, is: what happens when the system has more than one iscsi device? :)... but that's for later I guess